src/components/ApiSetup/index.tsx

```diff
@@ -0,0 +1,214 @@
+  }, [createConnectionString, resetField, watch]);
+
+  return (
+    <div>
+      <Title level={3}>{t("title")}</Title>
+
+      <Text>{t("text")}</Text>
+
+      <Form layout="vertical" id="api-setup">
+        <Input
+          control={control}
+          name="name"
+          defaultValue={initialValue.name}
+          label={getLabel("data_source")}
+          disabled
+        />
+
+        <Input
+          control={control}
+          name="connection"
+          label={getLabel("connect_via")}
+          size="large"
+          optionType="button"
+          options={connectionOptions}
+          defaultValue={CONNECTION_DEFAULT}
+          fieldType="radio"
+        />
+
+        <Row gutter={[16, 16]}>
+          {connectionData.map((f) => {
+            const name = f.name as keyof ApiSetupForm;
+            return (
+              <Col key={f.label} xs={24} sm={12}>
+                <Input
+                  control={control}
+                  name={name}
+                  defaultValue={initialValue?.[name]}
+                  fieldType={f.type}
+                  label={f.label}
+                  disabled
+                  suffix={
+                    <CopyIcon
+                      className={styles.icon}
+                      onClick={() =>
+                        navigator.clipboard.writeText(getValues(name) || "")
+                      }
+                    />
+                  }
+                />
+              </Col>
+            );
+          })}
+        </Row>
+
+        <div className={cn(styles.textareaWrapper, styles.label)}>
+          <Input
+            control={control}
+            defaultValue={createConnectionString()}
+            name="connection_string"
+            fieldType="textarea"
+            label={`${getLabel("connect_using")} ${
+              watch("connection") || CONNECTION_DEFAULT
+            }-client`}
+            disabled
+          />
+
+          <CopyIcon
+            className={cn(styles.icon, styles.textareaCopy)}
+            onClick={() =>
+              navigator.clipboard.writeText(
+                getValues("connection_string") || ""
+              )
+            }
+          />
+        </div>
+
+        <Row>
+          <Button
+            className={cn(styles.back, { [styles.sm]: !windowSize.sm })}
+            size="large"
+            color="primary"
+            onClick={onGoBack}
+          >
+            {t("common:words.back")}
+          </Button>
+          <Button
+            className={cn(styles.submit, { [styles.sm]: !windowSize.sm })}
+            type="primary"
+            size="large"
+            htmlType="submit"
+            form="api-setup"
+            onClick={handleSubmit(onSubmit)}
+          >
+            {t("common:words.finish")}
+          </Button>
+
+          <Button className={styles.link} type="link" onClick={onDownload}>
+            {t("common:words.download_credentials")}
+          </Button>
```

Comment:
is it possible to use `space` component here? https://ant.design/components/space